cypress/support/index.js

```diff
@@ -1,14 +1,65 @@
 import "./commands";
 
-Cypress.Commands.add("login", (email, password) => {
-    cy.visit("/login")
+Cypress.Commands.add(
+    "login",
+    (email = Cypress.env("email"), password = Cypress.env("password")) => {
+        cy.visit("/login")
 
-        .get("#email")
-        .type(email)
+            .get("#email")
+            .type(email)
 
-        .get("#password")
-        .type(password)
+            .get("#password")
+            .type(password)
 
-        .get("#submit")
-        .click();
+            .get("#submit")
+            .click();
+    }
+);
+
+Cypress.Commands.add("logout", () => {
+    cy.visit("/").get("#logout").click();
+});
+
+Cypress.Commands.add(
+    "addReview",
+    (
+        name = "Test Name",
+        description = "Test Description",
+        rating = 4,
+        fav = true,
+        tags = ["tag1", "tag2", "tag3"]
+    ) => {
+        cy.visit("/add");
+
+        if (name) {
+            cy.get("#name").type(name);
+        }
+
+        if (description) {
+            cy.get("#description").type(description);
+        }
+
+        if (rating) {
```

Comment:
This is going to fail if you pass `0` as a rating, because `0` is falsy in JavaScript, meaning it will not set the rating. You have a couple of ways to solve this:
```js
if (typeof rating !== 'undefined') {
```
```js
if (rating || rating === 0) {
```